templates/crate/details.html

```diff
@@ -0,0 +1,304 @@
+{%- extends "base.html" -%}
+{%- import "header/package_navigation.html" as navigation -%}
+
+{%- block title -%}
+    {{ macros::doc_title(name=details.name, version=details.version) }}
+{%- endblock title -%}
+
+{%- block header -%}
+    {# Set the active tab to the `crate` tab #}
+    {{ navigation::package_navigation(metadata=details.metadata, active_tab="crate") }}
+{%- endblock header -%}
+
+{%- block body -%}
+    <div class="container package-page-container">
+        <div class="pure-g">
+            <div class="pure-u-1 pure-u-sm-7-24 pure-u-md-5-24">
+                <div class="pure-menu package-menu">
+                    <ul class="pure-menu-list">
+                        {# List the release author's names and a link to their docs.rs profile #}
+                        <li class="pure-menu-heading">Authors</li>
+                        {%- for author in details.authors -%}
+                            <li class="pure-menu-item">
+                                <a href="/releases/{{ author[1] }}" class="pure-menu-link">
+                                    {{ author[0] }}
+                                </a>
+                            </li>
+                        {%- endfor -%}
+
+                        <li class="pure-menu-heading">Links</li>
+
+                        {# If the crate has a homepage, show it #}
+                        {%- if details.homepage_url -%}
```

Comment:
Can you give a screenshot of what the `details` page looks like when details is `None`?

Reply:
The homepage just isn't there